scalafmt-tests/src/test/resources/align/DefaultWithAlign.stat

```diff
@@ -1179,3 +1179,63 @@ libraryDependencies ++= Seq(
   "com.ibm"           %% "couchdb-scala" % "0.6.0.1" % "test" %% 123,
   "com.googlecode.java-diff-utils" % "diffutils" // c1
 )
+<<< align without parent
+align.tokens."+" = [{ code=":", owner="Term\\.Param" }]
+===
+object a {
+  case class Foo(
+    firstParam: Int,
+    secondParam: String,
+    thirdParam: Boolean
+  ) {
+    def Foo(
+      firstParam: Int,
+      secondParam: String,
+      thirdParam: Boolean
+    ) = ???
+  }
+}
+>>>
+object a {
```

Comment:
I think I don't really understand this settings, should the `:` be aligned?
```scala
object a {  
  case class Foo(
    firstParam : Int,      
    secondParam: String,      
    thirdParam : Boolean  
  ) { 
      def Foo(        
        firstParam : Int,        
        secondParam: String,        
        thirdParam : Boolean    
      ) = ???  
}}
```

This seems to work in case of `=>`, but in the [examples](https://scalameta.org/scalafmt/docs/configuration.html#aligntokens) it doesn't seem to align anything for `%%` or `%`

Reply:
But it actually work for `build.sbt` file I tested :thinking:

Reply:
Sure I was just thinking based on docs that it would align colons instead of the type after it (it does align `%`, `=>` and not the expression afterwards). If that is the intention then I misunderstood the setting.

As for the lack of `%` align i thought it was connected to the same issue, but seems to be just a problem with the docs themselves

Reply:
> Sure I was just thinking based on docs that it would align colons instead of the type after it (it does align `%`, `=>` and not the expression afterwards). If that is the intention then I misunderstood the setting.

the reason is not mentioned in the code but looks like a fix for #174. it only aligns if there's a space, and uses the previous alignment offset if necessary.